installation/scripts/testing.sh

```diff
@@ -223,7 +241,7 @@ testCheckGateway=$?
 printImagesWithLatestTag
 latestTagsErr=$?
 
-if [ ${latestTagsErr} -ne 0 ] || [ ${coreTestErr} -ne 0 ]  || [ ${istioTestErr} -ne 0 ] || [ ${acTestErr} -ne 0 ]
+if [ ${latestTagsErr} -ne 0 ] || [ ${coreTestErr} -ne 0 ]  || [ ${istioTestErr} -ne 0 ] || [ ${acTestErr} -ne 0 ] || [ ${loggingTestErr} -ne 0 ] || [ ${monitoringTestErr} -ne 0 ]
```

Comment:
in case of the minikube installation you will receive such error
```
./scripts/testing.sh: line 243: [: -ne: unary operator expected
./scripts/testing.sh: line 243: [: -ne: unary operator expected
```

because the `loggingTestErr` and `monitoringTestErr ` are not defined.

Reply:
fixed.